From 4a1ebf20c4d9d0916921f6b712a696348e32b122 Mon Sep 17 00:00:00 2001
From: Danny Kukawka <danny.kukawka@web.de>
Date: Thu, 30 Jul 2009 17:56:24 +0200
Subject: [PATCH 08/19] add a Creative MuVo 100 as a USB music player

Applied a fixed/slightly adopted version of a patch from
Reinhard Jentsch <pinswats@yahoo.se> (fd.o#22243) to add a
Creative MuVo 100 as a USB music player.
---
 .../10freedesktop/10-usb-music-players.fdi         |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/fdi/information/10freedesktop/10-usb-music-players.fdi b/fdi/information/10freedesktop/10-usb-music-players.fdi
index 55495f4..0e755c0 100644
--- a/fdi/information/10freedesktop/10-usb-music-players.fdi
+++ b/fdi/information/10freedesktop/10-usb-music-players.fdi
@@ -35,6 +35,11 @@
 
         <!-- Creative -->
 	<match key="@storage.originating_device:usb.vendor_id" int="0x41e">
+	  <!-- MuVo V100 -->
+          <match key="@storage.originating_device:usb.product_id" int="0x200c">
+	    <addset key="portable_audio_player.access_method.protocols" type="strlist">storage</addset>
+	    <append key="portable_audio_player.input_formats" type="strlist">audio/x-wav</append>
+	  </match>
 	  <!-- MuVo NX, MuVo2, MuVo TX, MuVo TX FM, Zen Micro,NOMAD Jukebox Zen Xtra, MuVo N200 (MuVo Micro), Zen Micro -->
 	  <match key="@storage.originating_device:usb.product_id" int_outof="0x4115;0x4116;0x4117;0x4120;0x411e;0x4128;0x412b;0x4130">
 	    <addset key="portable_audio_player.access_method.protocols" type="strlist">storage</addset>
-- 
1.6.4.4

